app/scripts/controllers/user-storage/encryption/encryption.ts

```diff
@@ -1,36 +1,45 @@
-import { pbkdf2 } from '@noble/hashes/pbkdf2';
+import { scrypt } from '@noble/hashes/scrypt';
 import { sha256 } from '@noble/hashes/sha256';
 import { utf8ToBytes, concatBytes, bytesToHex } from '@noble/hashes/utils';
 import { gcm } from '@noble/ciphers/aes';
 import { randomBytes } from '@noble/ciphers/webcrypto';
+import { getAnyCachedKey, getCachedKeyBySalt, setCachedKey } from './cache';
+import { base64ToByteArray, byteArrayToBase64, bytesToUtf8 } from './utils';
 
 export type EncryptedPayload = {
-  v: '1'; // version
-  d: string; // data
-  iterations: number;
-};
+  // version
+  v: '1';
 
-function byteArrayToBase64(byteArray: Uint8Array) {
-  return Buffer.from(byteArray).toString('base64');
-}
+  // encryption type - scrypt
```

Comment:
The comment should probably be something like `// key derivation function algorithm - scrypt`, "encryption type" doesn't look correct.